### PR TITLE
create workflow to publish javascript github actions

### LINKED ISSUE #79: create workflow to publish actions in v*-release branch
Use the recommended release workflow for javascript action:
https://github.com/actions/toolkit/blob/master/docs/javascript-action.md

### PR DESCRIPTION
**Pull request by vibou**
fixes #79

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 node_modules
```
Comment (shouze) [inline .gitignore]: Sure?

Comment (vibou) [reply .gitignore]: Nope. It's an artifact. I m glad review exists :)

Comment (vibou) [reply .gitignore]: Oh actually yes it has to be ignored in the master

Comment (vibou) [reply .gitignore]: No node_modules in master only in release branches. The github action take care of that ;)

**Pull request merged**